test/test_jit.py

```diff
@@ -0,0 +1,420 @@
+        @torch.jit.script
+        def jit_method(tensor, ch_dim, max_len, len_dim, fill_value):
+            # type: (Tensor, int, int, int, float) -> Tensor
+            return F.pad_trim(tensor, ch_dim, max_len, len_dim, fill_value)
+
+        tensor = torch.rand((10, 1))
+        ch_dim = 1
+        max_len = 5
+        len_dim = 0
+        fill_value = 3.
+
+        jit_out = jit_method(tensor, ch_dim, max_len, len_dim, fill_value)
+        py_out = F.pad_trim(tensor, ch_dim, max_len, len_dim, fill_value)
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
+
+    @unittest.skipIf(not RUN_CUDA, "no CUDA")
+    def test_scriptmodule_pad_trim(self):
+
+        class MyModule(torch.jit.ScriptModule):
+            def __init__(self, max_len):
+                super(MyModule, self).__init__()
+                self.module = transforms.PadTrim(max_len)
+                self.module.eval()
+
+            @torch.jit.script_method
+            def forward(self, tensor):
+                return self.module(tensor)
+
+        tensor = torch.rand((1, 10), device="cuda")
+        max_len = 5
+
+        jit_out = MyModule(max_len).cuda()(tensor)
+        py_out = transforms.PadTrim(max_len).cuda()(tensor)
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
+
+    def test_torchscript_downmix_mono(self):
+        @torch.jit.script
+        def jit_method(tensor, ch_dim):
+            # type: (Tensor, int) -> Tensor
+            return F.downmix_mono(tensor, ch_dim)
+
+        tensor = torch.rand((10, 1))
+        ch_dim = 1
+
+        jit_out = jit_method(tensor, ch_dim)
+        py_out = F.downmix_mono(tensor, ch_dim)
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
+
+    @unittest.skipIf(not RUN_CUDA, "no CUDA")
+    def test_scriptmodule_downmix_mono(self):
+
+        class MyModule(torch.jit.ScriptModule):
+            def __init__(self):
+                super(MyModule, self).__init__()
+                self.module = transforms.DownmixMono()
+                self.module.eval()
+
+            @torch.jit.script_method
+            def forward(self, tensor):
+                return self.module(tensor)
+
+        tensor = torch.rand((1, 10), device="cuda")
+
+        jit_out = MyModule().cuda()(tensor)
+        py_out = transforms.DownmixMono().cuda()(tensor)
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
+
+    def test_torchscript_LC2CL(self):
+        @torch.jit.script
+        def jit_method(tensor):
+            # type: (Tensor) -> Tensor
+            return F.LC2CL(tensor)
+
+        tensor = torch.rand((10, 1))
+
+        jit_out = jit_method(tensor)
+        py_out = F.LC2CL(tensor)
+
+        self.assertTrue(torch.allclose(jit_out, py_out, atol=5e-4, rtol=1e-4))
+
+    @unittest.skipIf(not RUN_CUDA, "no CUDA")
+    def test_scriptmodule_LC2CL(self):
+
+        class MyModule(torch.jit.ScriptModule):
```

Comment:
It might be possible to abstract this into a helper function that you give the specific transform and then returns the class.